### PR TITLE
Update qanda.json

### PR DESCRIPTION
**Pull request by ShlomoCode**
I added a Hebrew translation for the added strings

### CODE REVIEW

File: static/languages/he/qanda.json [hunk 1]
```diff
@@ -3,10 +3,19 @@
     "topic_unsolved": "לא נפתר",
     "menu.solved": "נפתר",
     "menu.unsolved": "לא נפתר",
+    "abel.solution": "פתרון",
```
Comment (barisusakli) [inline static/languages/he/qanda.json:6]: should be `label.solution`

File: static/languages/he/qanda.json [hunk 2]
```diff
@@ -17,5 +26,7 @@
     "admin.form.general_settings": "הגדרות כלליות",
     "admin.form.label.only_allow_all": "אפשר שאלות בלבד בכל הקטגוריות (בטל נושאים רגילים)",
     "admin.form.label.only_allow_following": "אפשר שאלות בלבד בקטגוריות המסומנות (בטל נושאים רגילים)",
-    "admin.form.tips": "הגדרה זו עוקפת את ההגדרה למטה."
+    "admin.form.tips": "הגדרה זו עוקפת את ההגדרה למטה.",
+    "admin.form.label.toggle_loc": "נעל אוטומטית נושאים שנפתרו",
```
Comment (barisusakli) [inline static/languages/he/qanda.json:30]: should be `admin.form.label.toggle_lock`

Comment (ShlomoCode) [reply static/languages/he/qanda.json]: Oops... thanks for checking, fixed

File: static/languages/he/qanda.json (hunk 1 above)
Comment (ShlomoCode) [reply static/languages/he/qanda.json]: Oops... thanks for checking, fixed

**Comment by barisusakli** [comment]
@ShlomoCode thanks, published 1.0.18

**Pull request merged**